src/pages/new-chat/new-chat.ts

```diff
@@ -41,28 +52,59 @@ export class NewChatComponent implements OnInit {
     });
   }
 
-  private findUsers(): ObservableCursor<User> {
+  findContacts(): Promise<Contact[]> {
+    // If we're running this in the browser, don't look for any contacts
+    if (!navigator.hasOwnProperty('contacts')) return Promise.resolve();
```

Comment:
I wanted to handle a use case where the `contacts` plugin is simply not installed, regardless of what platform your'e using. Your option seems cleaner and nice through.